src/getbits.rs

```diff
@@ -189,4 +189,14 @@ impl<'a> GetBits<'a> {
     pub const fn pos(&self) -> usize {
         self.index * u8::BITS as usize - self.bits_left as usize
     }
+
+    #[inline]
```

Comment:
```suggestion
```

We shouldn't need manual `#[inline]`s where C doesn't have them and looking at the assembly doesn't show a difference.  If it's a short `const fn`, it's basically `#[inline]` anyways, and in practice, `rustc` now makes all short functions `#[inline]` (that's why we need `#[inline(never)]` on godbolt for short functions).